src/main/java/stream/flarebot/flarebot/commands/secret/AnnounceCommand.java

```diff
@@ -0,0 +1,51 @@
+package stream.flarebot.flarebot.commands.secret;
+
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.entities.Role;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+
+public class AnnounceCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message msg, String[] args, Member member) {
+        if(guild.getGuildId().equals("226785954537406464")) {
+            Role r = guild.getGuild().getRoleById(320304080926801922L);
+            r.getManager().setMentionable(true).complete();
+            String message = msg.getRawContent();
+            message = message.substring(message.indexOf(" ") + 1);
+            guild.getGuild().getTextChannelById(226786449217945601L).sendMessage(r.getAsMention() + "\n" + message).complete();
+            r.getManager().setMentionable(false).complete();
+        }
+    }
+
+    @Override
+    public String getCommand() {
+        return "announce";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Dev only command";
+    }
+
+    @Override
+    public String getUsage() {
+        return "{%}announce [message]";
+    }
+
+    @Override
+    public CommandType getType() {
+        return CommandType.SECRET;
+    }
+
+    @Override
+    public boolean isDefaultPermission() {
```

Comment:
This isn't needed, it's already not default and this check isn't even fired due to it being a secret command.

Reply:
+1

Reply:
This exists in more files so I left it in.

Reply:
Still should be removed, the others will be as well when we get round to it.